pkg/k8sclient/k8spodwatcher.go

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
nit: 2019?

Reply:
Done.

Reply:
Actually changing it to 2019 was causing the verify-boilerplate script to fail, as it checks for year 2018 only.

Verifying verify-boilerplate.sh

+++ Running case: verify.boilerplate 
+++ working dir: /home/ubuntu/e2etest/src/github.com/kubernetes-sigs/poseidon
+++ command: bash "hack/make-rules/../../hack/verify-boilerplate.sh"
Boilerplate header is wrong for: /home/ubuntu/e2etest/src/github.com/kubernetes-sigs/poseidon/pkg/k8sclient/k8spodwatcher.go
+++ exit code: 1
+++ error: 1
FAILED   verify-boilerplate.sh	0s